sympy/polys/rings.py

```diff
@@ -2531,6 +2531,124 @@ def compose(f, x, a=None):
 
         return poly
 
+
+    def coeff_wrt(self, sym, dg):
+        """
+        Coefficient of ``f`` with respect to ``x**deg``.
+
+        Treating ``f`` as a univariate polynomial in ``x`` this finds the
+        coefficient of ``x**deg`` as a polynomial in the other generators.
+
+        Parameters
+        ==========
+
+        p : sympy.Poly
+            The polynomial to compute the sparse coefficient for.
+        sym : int
+            The symbol to compute the coefficient for.
```

Comment:
I think it should be possible to pass `sym` as the generator like with the `degree` method.

Reply:
Should I make `sym` optional?

Reply:
No. If it was optional then it would presumably default to doing something that some other method already does like `LC`.

Reply:
With the latest changes this does not need to be an int.

Reply:
it should `generator or generator index` ?